app/templates/content.hbs

```diff
@@ -1,9 +1,12 @@
-{{title model.title}}
+{{title node.title}}
 
 <div id="view-page">
     <div class="p-t-xl m-t-xl p-b-md m-b-md dark-overlay-header-background header-row">{{!HEADER ROW}}
         <div class="container">{{!CONTAINER DIV}}
-            <h1 id="preprintTitle" class="p-b-md">{{model.title}}</h1>
+            <h1 id="preprintTitle" class="p-b-md">{{node.title}}</h1>
+            {{#if isAdmin}}
+                <a type="button" class="pull-right btn btn-success m-h-lg" href={{concat model.links.html '?edit'}}  {{action "click" 'link' "Content - Edit Preprint" (concat model.links.html '?edit')}}>{{t "content.project_button.edit_preprint"}}</a>
```

Comment:
Again, from above, this issue was looked at today.  Moving forward as-is.  

"editing" preprints involves using the content route, but forcing the submit controller/template.  Having difficulties forcing setup, renderTemplate, and setupController to re-run with the query-param route.  Using <a> tag instead of link-to helper gets around this issue because it forces everything to reload.